docusaurus/docusaurus.config.js

```diff
@@ -129,7 +129,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve("./sidebars.js"),
```

Comment:
I didn't think about this when I added the expanded sidebars PR. You'll need to update this so that it doesn't overwrite.

```json
docs: {
          sidebarPath: require.resolve("./sidebars.js"),
          editUrl:
            "https://github.com/BobaBoard/boba-docs/edit/main/docusaurus/",
```

```json
docs: {
          sidebarPath: require.resolve("./sidebars.js"),
	  sidebarCollapsed: false,
          editUrl:
            "https://github.com/BobaBoard/boba-docs/edit/main/docusaurus/",
```

Sorry!

Reply:
I.. think I did that correctly. Hopefully.